.github/workflows/ci.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+env:
+  RUSTFLAGS: -Dwarnings
+  RUST_BACKTRACE: 1
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust
+        run: rustup update stable
+      - run: cargo test --all-features --workspace
+
+  no-std:
+    strategy:
+      matrix:
+        target:
+          # TODO: Currently, valuable cannot build with targets that do not have atomic CAS.
+          #       We should port https://github.com/rust-lang/futures-rs/pull/2400.
+          # - thumbv6m-none-eabi
```

Comment:
The fix of this is in my other branch (https://github.com/tokio-rs/valuable/commit/be24a36bffdde6e74234e753845c43c4ad0bd4d2), but it is bigger than this PR, so perhaps separate PR would be preferable.